apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -169,4 +169,33 @@ export class BookingsRepository_2024_08_13 {
       },
     });
   }
+
+  async getUserBookingsForTimeRange(userId: number, startTime: Date, endTime: Date) {
```

Comment:
First, the function is called `getUserBookingsForTimeRange` but it has `status: "ACCEPTED",` - so instead it should be called `getAcceptedUserBookingsForTimeRange`.

Second, it only gets 1 booking because of `findFirst`, which then is misleading if I read the function I expect it would return to me all bookings, so you could call it `getAcceptedUserBookingForTimeRange`.